ballerina/executor_visitor.bal

```diff
@@ -24,11 +24,13 @@ class ExecutorVisitor {
     private Data data;
     private ErrorDetail[] errors;
     private Context context;
+    private map<FileUpload|FileUpload[]> fileInfo;
 
-    isolated function init(Engine engine, __Schema schema, Context context) {
+    isolated function init(Engine engine, __Schema schema, Context context, map<FileUpload|FileUpload[]>? fileInfo) {
         self.engine = engine;
         self.schema = schema;
         self.context = context;
+        self.fileInfo = fileInfo == () ? {} : fileInfo;
```

Comment:
Can we get rid of this assignment by making the `fileInfo` a default-able input param which defaults to `{}`?